fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1535,6 +1549,12 @@ private String getAddSql(final List<ResultsetColumnHeaderData> columnHeaders, fi
                 columnName = sqlGenerator.escape(key);
                 insertColumns += ", " + columnName;
                 selectColumns += "," + pValueWrite + " as " + columnName;
+            } else {
+                if (key.equalsIgnoreCase("created_at") || key.equalsIgnoreCase("updated_at")) {
```

Comment:
Could we extract the created_at and updated_at strings into constants so they're not everywhere in the datatables codebase as magic strings?

Reply:
Solved